src/NzbDrone.Core/Notifications/Webhook/Webhook.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Core.Tv;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    public class Webhook : NotificationBase<WebhookSettings>
+    {
+        private readonly IWebhookService _service;
+
+        public Webhook(IWebhookService service)
+        {
+            _service = service;
+        }
+
+        public override string Link
+        {
+            get { return "https://github.com/Sonarr/Sonarr/wiki/Webhook"; }
```

Comment:
can you add a wiki page with sample requests that are sent by the webhook, example of the body and headers would be great, maybe mention that it's limited to `POST` and `JSON` for now,